ursula_cli/_manifest.py

```diff
@@ -0,0 +1,11 @@
+import yaml
+
+
+class Manifest(object):
+    def __init__(self, data):
+        self.ansible_version = data['ansible']['version']
```

Comment:
A better way to do this would be with *args. Something like:
```
def __init__(self, ansible_version):
    self.ansbile_version = ansible_version
```
and in load_file:
```
return UrsulaManifest(*dct['ursula']])
```